views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,4 @@
+<img src="{{image_url}}" alt="beer image">
```

Comment:
Sometimes the random beer does not contain an image URL, thus no image.
In the "/random-beer" route, the image_url property value can be set to a default image in case it is undefined.